tests/testthat/_snaps/dataframe.md

```diff
@@ -17,7 +17,7 @@
       │ max        ┆ 33.9      ┆ 8.0      ┆ 472.0      ┆ … ┆ 1.0      ┆ 1.0      ┆ 5.0      ┆ 8.0    │
       │ median     ┆ 19.2      ┆ 6.0      ┆ 196.3      ┆ … ┆ 0.0      ┆ 0.0      ┆ 4.0      ┆ 2.0    │
       │ 25pct      ┆ 15.5      ┆ 4.0      ┆ 121.0      ┆ … ┆ 0.0      ┆ 0.0      ┆ 3.0      ┆ 2.0    │
-      │ 75pct      ┆ 22.8      ┆ 8.0      ┆ 350.0      ┆ … ┆ 1.0      ┆ 1.0      ┆ 4.0      ┆ 4.0    │
+      │ 75pct      ┆ 22.8      ┆ 8.0      ┆ 318.0      ┆ … ┆ 1.0      ┆ 1.0      ┆ 4.0      ┆ 4.0    │
```

Comment:
This is a side note, but I noticed that it shows a different value than the quontile of R.
Is there any possibility to improve upstream on how it is calculated?

Reply:
I opened an upstream issue pola-rs/polars#13480